lib/templates/bootstrap/datepicker.android.js

```diff
@@ -85,6 +85,11 @@ function datepicker(locals) {
     <Text style={dateValueStyle}>{formattedValue}</Text>
   ) : null;
 
+  var is24Hour = true;
+  if (typeof locals.config.is24Hour !== "undefined") {
```

Comment:
Why not `locals.config.is24Hour` directly?

Also, `is24Hour` should be added into the docs.

Reply:
I see, the check is not inside https://github.com/danyalsiraj/tcomb-form-native/blob/9df6e4fcc9711e9e1d3668d694720366f18ad8fd/lib/templates/bootstrap/datepicker.android.js#L54

Can you move it there in order to match the coding style?